actions/ostree_commit_action.go

```diff
@@ -1,3 +1,28 @@
+/*
+OstreeCommit Action
+
+Create OSTree commit from rootfs prepared with other actions.
+
+Yaml syntax:
+ - action: ostree-commit
+   repository: repository name
+   branch: branch name
+   subject: commit message
+
+Mandatory properties:
+
+- repository -- path to repository with OSTree structure; the same path is
+used by 'ostree' tool with '--repo' argument.
+This path is relative to 'artifact' directory.
+Please keep in mind -- you will need a root privileges for 'bare' repository
```

Comment:
May make sense to recommend the archive-z2 type ? I don't really see a use-case for using bare repositories here

Reply:
Not sure. But my first attempt fails because of mentioned problem. I'll be happy to see the warning instead of recommendation before my test ;)

Reply:
Well maybe just document that one should use the archive-z2 archive type in that case as this seems to indicate you'd have to have root. Can be done later though